hw_monitor:
  ros__parameters:
    polling_frequency: 2.0
    cpu : true
    memory: true
    disk : true
    network : true
    power : true
    temperature : true
